internal/utils/test_utilities.go

```diff
@@ -0,0 +1,5 @@
+package utils
+
+func PtrToStr(s string) *string {
```

Comment:
Change this name to `String` and filename to `ptr.go`. Actually  this method name `String` follows conventions from for example Jeff's Amazon Web Services SDK. See https://pkg.go.dev/github.com/aws/aws-sdk-go-v2@v1.26.1/aws#String